Build fails: member reference type 'std::istream *' (aka 'basic_istream<char> *') is a pointer; did you mean to use '->'

```
/usr/ports/cad/cascade-compiler/work/.build/verilog_lexer.cc:1158:8: error: member reference type 'std::istream *' (aka 'basic_istream<char> *') is a pointer; did you mean to use '->'?
                        yyin.rdbuf(std::cin.rdbuf());
                        ~~~~^
                            ->
/usr/ports/cad/cascade-compiler/work/.build/verilog_lexer.cc:1165:9: error: member reference type 'std::ostream *' (aka 'basic_ostream<char> *') is a pointer; did you mean to use '->'?
                        yyout.rdbuf(std::cout.rdbuf());
                        ~~~~~^
                             ->
/usr/ports/cad/cascade-compiler/work/.build/verilog_lexer.cc:2418:50: error: member reference type 'std::istream *' (aka 'basic_istream<char> *') is a pointer; did you mean to use '->'?
                        YY_CURRENT_BUFFER_LVALUE->yy_input_file = yyin.rdbuf();
                                                                  ~~~~^
                                                                      ->
/usr/ports/cad/cascade-compiler/work/.build/verilog_lexer.cc:2537:2: error: cannot initialize a member subobject of type 'std::istream *' (aka 'basic_istream<char> *') with an rvalue of type 'basic_streambuf<std::__1::basic_ios<char, std::__1::char_traits<char> >::char_type, std::__1::basic_ios<char, std::__1::char_traits<char> >::traits_type> *' (aka 'basic_streambuf<char, std::__1::char_traits<char> > *')
        yyin(arg_yyin ? arg_yyin->rdbuf() : std::cin.rdbuf()),
        ^    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/usr/ports/cad/cascade-compiler/work/.build/verilog_lexer.cc:2538:2: error: cannot initialize a member subobject of type 'std::ostream *' (aka 'basic_ostream<char> *') with an rvalue of type 'basic_streambuf<std::__1::basic_ios<char, std::__1::char_traits<char> >::char_type, std::__1::basic_ios<char, std::__1::char_traits<char> >::traits_type> *' (aka 'basic_streambuf<char, std::__1::char_traits<char> > *')
        yyout(arg_yyout ? arg_yyout->rdbuf() : std::cout.rdbuf())
        ^     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/usr/ports/cad/cascade-compiler/work/.build/verilog_lexer.cc:2540:2: error: use of undeclared identifier 'ctor_common'
        ctor_common();
        ^
/usr/ports/cad/cascade-compiler/work/.build/verilog_lexer.cc:2545:14: error: out-of-line definition of 'yyFlexLexer' does not match any declaration in 'yyFlexLexer'
yyFlexLexer::yyFlexLexer( std::istream& arg_yyin, std::ostream& arg_yyout ):
             ^~~~~~~~~~~
/usr/ports/cad/cascade-compiler/work/.build/verilog_lexer.cc:30:25: note: expanded from macro 'yyFlexLexer'
    #define yyFlexLexer yyFlexLexer
                        ^~~~~~~~~~~
/usr/ports/cad/cascade-compiler/work/.build/verilog_lexer.cc:2536:27: note: type of 1st parameter of member declaration does not match definition ('std::istream *' (aka 'basic_istream<char> *') vs 'std::istream &' (aka 'basic_istream<char> &'))
yyFlexLexer::yyFlexLexer( std::istream* arg_yyin, std::ostream* arg_yyout ):
                          ^
/usr/ports/cad/cascade-compiler/work/.build/verilog_lexer.cc:2546:2: error: cannot initialize a member subobject of type 'std::istream *' (aka 'basic_istream<char> *') with an rvalue of type 'basic_streambuf<std::__1::basic_ios<char, std::__1::char_traits<char> >::char_type, std::__1::basic_ios<char, std::__1::char_traits<char> >::traits_type> *' (aka 'basic_streambuf<char, std::__1::char_traits<char> > *')
        yyin(arg_yyin.rdbuf()),
        ^    ~~~~~~~~~~~~~~~~
/usr/ports/cad/cascade-compiler/work/.build/verilog_lexer.cc:2547:2: error: cannot initialize a member subobject of type 'std::ostream *' (aka 'basic_ostream<char> *') with an rvalue of type 'basic_streambuf<std::__1::basic_ios<char, std::__1::char_traits<char> >::char_type, std::__1::basic_ios<char, std::__1::char_traits<char> >::traits_type> *' (aka 'basic_streambuf<char, std::__1::char_traits<char> > *')
        yyout(arg_yyout.rdbuf())
        ^     ~~~~~~~~~~~~~~~~~
/usr/ports/cad/cascade-compiler/work/.build/verilog_lexer.cc:2549:2: error: use of undeclared identifier 'ctor_common'
        ctor_common();
        ^
/usr/ports/cad/cascade-compiler/work/.build/verilog_lexer.cc:2554:19: error: out-of-line definition of 'ctor_common' does not match any declaration in 'yyFlexLexer'
void yyFlexLexer::ctor_common()
                  ^~~~~~~~~~~
/usr/ports/cad/cascade-compiler/work/.build/verilog_lexer.cc:2592:19: error: out-of-line definition of 'switch_streams' does not match any declaration in 'yyFlexLexer'
void yyFlexLexer::switch_streams( std::istream& new_in, std::ostream& new_out )
                  ^~~~~~~~~~~~~~
/usr/include/FlexLexer.h:129:31: note: type of 1st parameter of member declaration does not match definition ('std::istream *' (aka 'basic_istream<char> *') vs 'std::istream &' (aka 'basic_istream<char> &'))
        virtual void switch_streams( FLEX_STD istream* new_in, FLEX_STD ostream* new_out = 0 );
                                     ^
/usr/include/FlexLexer.h:53:22: note: expanded from macro 'FLEX_STD'
#    define FLEX_STD std::
                     ^
/usr/ports/cad/cascade-compiler/work/.build/verilog_lexer.cc:2596:41: error: no viable conversion from 'std::istream' (aka 'basic_istream<char>') to 'std::istream *' (aka 'basic_istream<char> *')
        yy_switch_to_buffer( yy_create_buffer( new_in, YY_BUF_SIZE  ) );
                                               ^~~~~~
/usr/include/FlexLexer.h:121:62: note: passing argument to parameter 's' here
        struct yy_buffer_state* yy_create_buffer( FLEX_STD istream* s, int size );
                                                                    ^
/usr/ports/cad/cascade-compiler/work/.build/verilog_lexer.cc:2599:7: error: member reference type 'std::ostream *' (aka 'basic_ostream<char> *') is a pointer; did you mean to use '->'?
        yyout.rdbuf(new_out.rdbuf());
        ~~~~~^
             ->
/usr/ports/cad/cascade-compiler/work/.build/verilog_lexer.cc:2607:12: error: assigning to 'std::istream *' (aka 'basic_istream<char> *') from incompatible type 'std::istream **' (aka 'basic_istream<char> **'); remove &
                new_in = &yyin;
                         ^~~~~
/usr/ports/cad/cascade-compiler/work/.build/verilog_lexer.cc:2611:13: error: assigning to 'std::ostream *' (aka 'basic_ostream<char> *') from incompatible type 'std::ostream **' (aka 'basic_ostream<char> **'); remove &
                new_out = &yyout;
                          ^~~~~~
/usr/ports/cad/cascade-compiler/work/.build/verilog_lexer.cc:2614:17: error: no viable conversion from 'std::istream' (aka 'basic_istream<char>') to 'std::istream *' (aka 'basic_istream<char> *')
        switch_streams(*new_in, *new_out);
                       ^~~~~~~
/usr/ports/cad/cascade-compiler/work/.build/verilog_lexer.cc:2604:49: note: passing argument to parameter 'new_in' here
void yyFlexLexer::switch_streams( std::istream* new_in, std::ostream* new_out )
                                                ^
/usr/ports/cad/cascade-compiler/work/.build/verilog_lexer.cc:2623:11: error: member reference type 'std::istream *' (aka 'basic_istream<char> *') is a pointer; did you mean to use '->'?
        if ( yyin.eof() || yyin.fail() )
             ~~~~^
                 ->
/usr/ports/cad/cascade-compiler/work/.build/verilog_lexer.cc:2623:25: error: member reference type 'std::istream *' (aka 'basic_istream<char> *') is a pointer; did you mean to use '->'?
        if ( yyin.eof() || yyin.fail() )
                           ~~~~^
                               ->
fatal error: too many errors emitted, stopping now [-ferror-limit=]
```

bison-3.4.1,1
FreeBSD 12

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -4,8 +4,8 @@ project(cascade)
 set(CMAKE_CXX_STANDARD 17)
 set(CMAKE_CXX_STANDARD_REQUIRED ON)
 
-find_package(FLEX 2.6.0)
-find_package(BISON)
+find_package(FLEX 2.6.0 REQUIRED)
+find_package(BISON 3.2.0 REQUIRED)
 
 if (APPLE)
     set(cxxlib libc++)

--- a/README.md
+++ b/README.md
@@ -836,7 +836,9 @@ FAQ
 #### Flex fails during build with an error related to yyin.rdbuf(std::cin.rdbuf()) on OSX.
 This is related to the version of flex that you have installed; some versions
 of port will install an older revision. Try using the version of flex provided
-by XCode in ```/usr/bin/flex```.
+by XCode in ```/usr/bin/flex```. CMake should now detect older versions of flex
+and abort compilation before it gets to this point. If you see this error
+message, please contact the developers.
 
 #### Cascade emits strange warnings whenever I declare a module.
 Module declarations are typechecked in the global scope, separate from the rest

